spec/controllers/coronavirus_form/basic_care_needs_controller_spec.rb

```diff
@@ -26,13 +26,13 @@
 
     it "validates any option is chosen" do
       post :submit, params: { basic_care_needs: "" }
-      expect(response).to have_http_status(:unprocessable_entity)
+
       expect(response).to render_template(current_template)
     end
 
     it "validates a valid option is chosen" do
       post :submit, params: { basic_care_needs: "<script></script>" }
-      expect(response).to have_http_status(:unprocessable_entity)
```

Comment:
Does render templat imply a 200 or can we do `to_have_http_status(:success)` or something similar?